modules/cluster-limitations-local-zone.adoc

```diff
@@ -17,3 +17,14 @@ The following list details limitations when deploying a cluster in AWS Local Zon
 - Network resources such as Network Load Balancer (NLB), Classic Load Balancer, and Network Address Translation (NAT) Gateways are not supported in AWS Local Zones.
 - For an {product-title} cluster on AWS, the AWS Elastic Block Storage (EBS) `gp3` type volume is the default for node volumes and the default for the storage class. This volume type is not globally available on Local Zone locations. By default, the nodes running in Local Zones are deployed with the `gp2` EBS volume. The `gp2-csi` `StorageClass` must be set when creating workloads on Local Zone nodes.
 ====
+
+If you want the installation program to automatically create Local Zone subnets for your {product-title} cluster, specific configuration limitations apply with this method.
+
+[IMPORTANT]
+====
+The following list details configuration limitations that apply when you set the installation program to automatically create subnets for your {product-title} cluster:
+
+- A network plugin, which is specific to an overlay network, automatically adjusts the MTU value for an overlay network. With the installation program automatically creating subnets on your behalf, you cannot adjust the MTU value for the overlay network.
```

Comment:
I see that your PR mentions a way that a user can adjust the MTU value for an overlay network. Should I document this scenario?

https://github.com/openshift/installer/pull/7137/files#diff-d4da2968a241c2898ad36d96733640d0392692a1c7c1941a0a80e120a4f0af32R29-R69:~:text=Adjust%20the%20MTU%20of%20the%20overlay%20network

Reply:
No, we are not covering this manual change in 4.13+. The installer will automatically do it, if we'll need to cover it it must be added as a day-2 ops, discussed in the channel but not directions. https://redhat-internal.slack.com/archives/C054W7GB7HV/p1682695753357869
The steps are documented in the KCS https://access.redhat.com/solutions/6996487